frontend/src/modules/entitydetails/components/Metadata.js

```diff
@@ -167,19 +167,16 @@ export default class Metadata extends React.Component<Props> {
             { this.renderSources(entity) }
             <Localized id='entitydetails-Metadata--resource' attrs={ { title: true } }>
                 <Property title='Resource' className='resource'>
+                    <a href={ `/${locale.code}/${entity.project.slug}/` }>
+                        { entity.project.name }
+                    </a>
                     <a
```

Comment:
Sorry for not bringing this up earlier: could you instead of using `::before` pseudoclass add a delimiter `<span>` element? Right now the middot is clickable as well (it's part of the resource link), but it shouldn't be.

Reply:
Yes you are correct.